packages/block-editor/src/components/link-control/README.md

```diff
@@ -44,6 +44,19 @@
 - Type: `Function`
 - Required: No
 
+Use this callback as an opportunity to know if the link has been changed because of user edition.
```

Comment:
How about "Use this callback to take an action after a user set or updated a link."